Makefile

```
@@ -48,20 +48,18 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 
 quality: ## check coding style with pycodestyle and pylint
 	touch tests/__init__.py
-	pylint openedx-atlas tests test_utils manage.py *.py
+	pylint openedx_atlas tests *.py
 	rm tests/__init__.py
-	pycodestyle openedx-atlas tests  *.py
-	pydocstyle openedx-atlas tests *.py
-	isort --check-only --diff --recursive tests test_utils openedx-atlas *.py test_settings.py
+	pycodestyle openedx_atlas tests  *.py
+	pydocstyle openedx_atlas tests *.py
+	isort --check-only --diff tests openedx-atlas *.py test_settings.py
 	python setup.py bdist_wheel
 	twine check dist/*
-	make selfcheck
-
 
 requirements: ## install development environment requirements
 	pip install -r requirements/pip.txt
 	pip install -r requirements/pip-tools.txt
-	pip-sync requirements/dev.txt requirements/private.*
+	pip-sync requirements/dev.txt
 
 test: clean ## run tests in the current virtualenv
 	pytest
```

```suggestion
		pip install -e .
		pytest
```

We need to install the current app or it won't work